inaugurator/partitiontable.py

```diff
@@ -271,23 +271,24 @@ def _isPartitionOfPhysicalDevice(self, device):
 
     @classmethod
     def getDevicesWithLabel(self, label):
-        RETRIES = 5
-        for retry in xrange(RETRIES):
-            os.system("/usr/sbin/busybox mdev -s")
-            time.sleep(1)
-            output = sh.run("blkid")
-            print "blkid output:\n"
-            print output
-            for line in output.splitlines():
-                line = line.strip()
-                parts = line.split(":", 1)
-                if len(parts) != 2:
-                    continue
-                device, data = parts
-                if " LABEL=\"%s\"" % label in data:
-                    device = device.strip()
-                    yield device
-            print "didn't find device with label %s. retry %d" % (label, retry)
+        os.system("/usr/sbin/busybox mdev -s")
+        time.sleep(1)
+        output = sh.run("blkid")
+        print "blkid output:\n"
```

Comment:
why do u have print here and logging above? @michaelk-stratoscale what was our conclusion about it?